src/Emitter/SapiEmitter.php

```diff
@@ -10,6 +10,13 @@
 final class SapiEmitter implements EmitterInterface
 {
     private const NO_BODY_RESPONSE_CODES = [204, 205, 304];
+    private $bufferSize;
+    private const DEFAULT_BUFFER_SIZE = 8388608; // 8MB
```

Comment:
@samdark Do we have a clear order of properties? It looks a little weird

Reply:
Oh, right. Constants first.

Reply:
Is it possible to create tests for this like changelog.md in yii2?

Reply:
Yes, it is possible.

Reply:
For now I've updated https://github.com/yiisoft/docs/blob/master/010-code-style.md#property-constant-and-method-placement to be clear about this case.